src/fontra/views/editor/panel-transformation.js

```diff
@@ -296,7 +311,121 @@ export default class TransformationPanel extends Panel {
               "flip horizontally"
             ),
         }),
-      ],
+      },
+    });
+
+    formContents.push({ type: "spacer" });
+    formContents.push({ type: "header", label: `Align Objects` });
+
+    formContents.push({
+      type: "universal-row",
+      field1: {
+        type: "auxiliaryElement",
+        key: "AlignLeft",
+        auxiliaryElement: html.createDomElement("icon-button", {
+          "src": "/tabler-icons/vertical-align-left.svg",
+          "onclick": (event) => this.moveObjects(alignLeft), //this._alignObjects("align left"),
```

Comment:
Some leftover comments (also in the next section).

Reply:
Sorry. Done.